src/mono/wasm/Wasm.Build.Tests/TestAppScenarios/AppTestBase.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Authentication.ExtendedProtection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Playwright;
+using Xunit.Abstractions;
+
+namespace Wasm.Build.Tests.TestAppScenarios
```

Comment:
Please use file-scoped namespaces.
```suggestion
namespace Wasm.Build.Tests.TestAppScenarios;
```